src/sonic-config-engine/minigraph.py

```diff
@@ -1142,7 +1142,26 @@ def parse_spine_chassis_fe(results, vni, lo_intfs, phyport_intfs, pc_intfs, pc_m
 #
 ###############################################################################
 
-def filter_acl_table_bindings(acls, neighbors, port_channels, sub_role):
+def filter_acl_table_for_backend(acls, vlan_members):
+    filter_acls = {}
+    for acl_name in acls:
```

Comment:
>acls [](http://example.com/codeflow?start=20&length=4)

you can get key and value at the same time `in acls.items()`.

Reply:
Done